easy-example-experiment.py

```diff
@@ -22,17 +22,6 @@ def main():
     datasets = dm.select()
 
     algorithms = [
-        Algorithm(
-            name="COF",
-            main=DockerAdapter(image_name="registry.gitlab.hpi.de/akita/i/cof", skip_pull=True),
-            param_config=FixedParameters({
-                "n_neighbors": 20,
-                "random_state": 42
-            }),
-            data_as_file=True,
-            training_type=TrainingType.UNSUPERVISED,
-            input_dimensionality=InputDimensionality.MULTIVARIATE
-        ),
```

Comment:
Please add the `cof()`-function call instead of the manual definition of the COF-algorithm. If you merge `main` first, it is already there as a comment.